00_Intro/bedrock_boto3_setup.ipynb

```diff
@@ -632,13 +632,15 @@
     "\n",
     "Use text embeddings to convert text into meaningful vector representations. You input a body of text \n",
     "and the output is a (1 x n) vector. You can use embedding vectors for a wide variety of applications. \n",
-    "Bedrock currently offers one model for text embedding that supports text similarity (finding the \n",
+    "Bedrock currently offers Titan Embeddings for text embedding that supports text similarity (finding the \n",
     "semantic similarity between bodies of text) and text retrieval (such as search).\n",
-    "For the text embeddings model, the input text size is 512 tokens and the output vector length is 4096.\n",
+    "\n",
+    "At the time of writing you can use `amazon.titan-embed-g1-text-02` as embedding model via the API. The input text size is 512 tokens and the output vector length is 4096.\n",
```

Comment:
Incorrect input/output size. The v2 model uses 8192 and 1536, respectively.

Also, do you think we should list both the v1 and v2 models?
```suggestion
    "At the time of writing you can use `amazon.titan-embed-g1-text-02` as embedding model via the API. The input text size is 8192 tokens and the output vector length is 1536.\n",
```

Reply:
Thanks for feedback. Corrected this.